src/common/time/Cargo.toml

```diff
@@ -12,6 +12,7 @@ common-error = { path = "../error" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = { version = "0.7", features = ["backtraces"] }
+greptime-proto.workspace = true
```

Comment:
question: Is it a good idea that `common-time` crate depends on `greptime-proto`? @v0y4g3r 

We could implement helper functions for conversion in `api::helper` mod like other pb types.

Reply:
thx! I know my problem now, I will try to refator it.